PORT-4483 logging for empty configuration (#110)
port-labs/ocean: changelog/PORT-4483.improvement.md

> Changed the handle of empty port app config in the PortAppConfig Handler

Reviewer: ```suggestion
Changed the way an empty port app config is handled in the `PortAppConfig Handler`
```
